app/controllers/books_controller.rb

```diff
@@ -46,6 +52,6 @@ def set_book
 
     # Only allow a trusted parameter "white list" through.
     def book_params
-      params.require(:book).permit(:name, :author, :sDescription, :lDescription, :image, :price, :amount)
+      params.require(:book).permit(:name, :author, :short_description, :long_description, :image_url, :price, :amount)
```

Comment:
talked about functional names, short_description and long_description are better but don't tell me much about the description.